posttowebhooks.go

```diff
@@ -0,0 +1,178 @@
+package main
+
+import (
+	"bytes"
+	"io"
+	"log"
+	"net/http"
+	"strings"
+	"text/template"
+
+	"github.com/buger/jsonparser"
+	"github.com/streadway/amqp"
+)
+
+//slack template
+//const templatetext = `
+//{
+//	"text": "{{.Msg}}. {{if .Completed}} <{{.Link}}|{{.LinkText}}> {{- end}}"
+//}
+//`
```

Comment:
This can probably be deleted.

Reply:
I just left here for testing template changes before updating the template in the database.